src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs

```diff
@@ -25,7 +26,7 @@ public class ServiceManagerBuilder : IServiceManagerBuilder, IDisposable
         /// <returns>The same instance of the <see cref="ServiceManagerBuilder"/> for chaining.</returns>
         public ServiceManagerBuilder WithOptions(Action<ServiceManagerOptions> configure)
         {
-            _services.Configure(configure);
+            _configure = configure;
```

Comment:
the behavior changes for `builder.WithOptions().WithOptions()...`

Reply:
The origin version before this pull request is [Here](https://github.com/Azure/azure-signalr/blob/7f3fdbc57ae3fab8772cb0b66e672bea0e77b7ab/src/Microsoft.Azure.SignalR.Management/ServiceManagerBuilder.cs#L25). The commit fixes the behavior change.